common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -60,7 +60,7 @@ steps:
   - script: node common/scripts/install-run-rush.js audit
     displayName: rush audit
     workingDirectory: ${{ parameters.workingDir }}
-    condition: and(succeeded(), ne(variables['Agent.OS'], 'Darwin'))
+    condition: or(eq(variables['Build.SourceBranch'],'release'), eq(variables['Build.SourceBranch'],'master'))
```

Comment:
- should be checking target branch not source branch if a pr
- should still be able to run on master and on demand like before
- should check if target branch contains `release/` not check if eq